src/mocks/handlers.js

```diff
@@ -53,6 +53,10 @@ export const handlers = [
   rest.post("/contact", (req, res, ctx) => {
     return res(ctx.status(200), ctx.json({ success: true }));
   }),
+  rest.post("/profile/:profileId", (req, res, ctx) => {
+    const body = req.body;
```

Comment:
Make the check if data are sent correctly and not empty, if empty, throw an error message